metadata-ingestion/src/datahub/ingestion/source/sql/oracle.py

```diff
@@ -299,7 +336,7 @@ def get_columns(
                 computed = None
 
             if identity_options is not None:
-                identity = self._inspector_instance.dialect._parse_identity_options(identity_options, default_on_nul)
+                identity = self.parse_identity_options(identity_options, default_on_nul)
```

Comment:
Motivation behind this change is not clear . Can you help me understand why this change is needed ?